src/hooks/useSubaccount.tsx

```diff
@@ -746,7 +746,15 @@ const useSubaccountContext = ({ localDydxWallet }: { localDydxWallet?: LocalWall
         throw new Error('wallet not initialized');
       }
 
-      const response = await compositeClient.createMarketPermissionless(subaccountClient, ticker);
+      track(AnalyticsEvents.LaunchMarketTransaction({ marketId: ticker }));
```

Comment:
will errors thrown by this function be logged to datadog or anything?

Reply:
Yeah, it will be logged to bugsnag

Reply:
datadog is built into the `log` function so it'll be sent to both

Reply:
<img width="475" alt="Screen Shot 2024-10-30 at 9 41 44 AM" src="https://github.com/user-attachments/assets/f27fc0bb-bf16-4f3e-99a9-e514f9081f23">

The dispatch will trigger bugsnag and the dd.error will also be sent